src/modules/Shared/components/NavbarPortal/NavbarPortal.tsx

```diff
@@ -104,16 +106,14 @@ export default function NavbarPortal(props: Props) {
             </Typography>
             <Box sx={{ display: { xs: "none", md: "block" }, me: 4 }}>
               <StyledNavLink
-               isActive={location.pathname === "/dashboard/homepage"}
-
+                isActive={location.pathname === "/dashboard/homepage"}
```

Comment:
No need for this prop. The NavLink component adds a class of `active` when a link is being matched. It also makes sure that the link is active even when the pathname is captalized or small letters. 
for example: "/DashBoard/HomePage" should be also a valid path to "/dashboard/homepage".

We can select the .active class in the style component api and add the styles for it.